bears/xml/XMLBear.py

```diff
@@ -15,8 +36,19 @@ def process_output(self, output, filename, file):
         if self.stderr_output:  # pragma: no cover
             return self._process_issues(self.stderr_output, filename)
 
-    def run(self, filename, file):
+    def run(self, filename, file,
+            schema: path="",
+            dtdvalid: path_or_url(path, str)=""):
```

Comment:
We probably want this to be names `xml_dtd` or so, as dtdvalid is not a very good name.
Even the above `schema` should be `xml_schema` or so ?